Corrigir erro nos testes

O teste abaixo está falhando, precisamos resolvê-lo!

```
FAIL: test_create_dict_blog (radar_parlamentar.tests.tests_blog.BlogTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/diraol/devel/radar/radar_parlamentar/radar_parlamentar/tests/tests_blog.py", line 10, in test_create_dict_blog
    self.assertEquals(blog_title,'PoliGNU - Grupo de Estudos de Software Livre da Poli-USP - Radar Parlamentar')
AssertionError: u'PoliGNU - Radar Parlamentar' != 'PoliGNU - Grupo de Estudos de Software Livre da Poli-USP - Radar Parlamentar'
```

--- a/radar_parlamentar/radar_parlamentar/tests/tests_blog.py
+++ b/radar_parlamentar/radar_parlamentar/tests/tests_blog.py
@@ -7,5 +7,5 @@ class BlogTest(TestCase):
     def test_create_dict_blog(self):
         dict_blog = DictionaryBlogGenerator.create_dict_blog()
         blog_title = dict_blog.feed.title
-        self.assertEquals(blog_title,'PoliGNU - Grupo de Estudos de Software Livre da Poli-USP - Radar Parlamentar')
+        self.assertEquals(blog_title, 'PoliGNU - Radar Parlamentar')
 

